### ISSUE TITLE
The result of Nerual ODE

### ISSUE THREAD

**Issue by marshall-dteach** [CarlosJose126/NeuralODE-ROM#2]
I have run your origin code, but the reconstruction result of `u_x` at `t=300` is not good as yours. Could you can give me some advices about how  to adjust to achieve the same results as yours? Thanks!

**Comment by marshall-dteach** [comment]
![image](https://user-images.githubusercontent.com/63546191/227157116-5e2c1ba8-921b-4779-976f-0c95a8fc5a04.png)
This is my reconstruction result.

**Comment by CarlosJose126** [comment]
Hi,

My suggestion is to retrain the models and find a new hyperparameter configuration using the random search that is implemented at the end of the scripts (VKS_lstm.py and VKS_node.py). Once you have trained and validated the models, you can evaluate and compare the ROM with the FOM.

Another option is to fine tune only the learning rate and keep the other hyperparameters fixed. You can use the train and validation losses to see if your model is overfitting. For example, try reducing the learning rate using a factor of 0.75 and retrain your model. 

I uploaded a file with a trained NODE model in my machine (see README), check if you get better results with this.

Best regards,
Carlos.
